packages/frontend-plugin-api/src/analytics/useAnalytics.test.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { renderHook } from '@testing-library/react';
+import { useAnalytics } from './useAnalytics';
+import { useApi } from '@backstage/core-plugin-api';
+
+jest.mock('@backstage/core-plugin-api', () => ({
```

Comment:
The useApi was mocked like this before. Would we like to refactor it to use a `TestApiProvider` wrapper?

Reply:
Yep that sounds better but no biggie.

Reply:
Done https://github.com/backstage/backstage/pull/21244/commits/3e7b765fbd59d325641e4f194d199028be67f7b6